### PR TITLE
refs #27928 - Fixing bad textation

### PR DESCRIPTION
**Pull request by domitea**

### CODE REVIEW

File: app/views/users/_form.html.erb [hunk 1]
```diff
@@ -88,7 +88,7 @@
     <% end %>
 
     <div class='tab-pane' id='roles'>
-      <% caption = @user.inherited_admin? ? "#{_('Admin rights are currently inherited from')} #{_('Usergroup')}" : '' %>
+      <% caption = @user.inherited_admin? ? "#{_('Admin rights are currently inherited from a user group')}" : '' %>
```
Comment (ekohl) [inline app/views/users/_form.html.erb]: Is the string interpolation still needed?
```suggestion
      <% caption = @user.inherited_admin? ? _('Admin rights are currently inherited from a user group') : '' %>
```

**Comment by ekohl** [comment]
Could you squash the commits? I'm also wondering how this relates to the Redmine issue. Looks like a fairly standalone issue.

**Comment by domitea** [comment]
> Could you squash the commits? I'm also wondering how this relates to the Redmine issue. Looks like a fairly standalone issue.

Sorry, 27298 and 27928 are very easy to write badly. Now It's should be squashed and issue number is correct

**Comment by ezr-ondrej** [comment]
There is still an merge commit present in your branch. That should be squashed (or removed) as well. Are you squashing using `git rebase -i <upstream>/develop`?

**Comment by ares** [comment]
@ekohl to answer your question, this was missed in the original PR review and is a follow up fix, that's wjy refs...

**Review by ekohl — APPROVED** [review]
Thanks! With the correct issue number it does make sense.

Test failure looks unrelated.

**Pull request merged**